manifests/m/Microsoft/FRSDiag/1.7/Microsoft.FRSDiag.locale.en-US.yaml

```
@@ -0,0 +1,17 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.12.0.schema.json
+
+PackageIdentifier: Microsoft.FRSDiag
+PackageVersion: "1.7"
+PackageLocale: en-US
+Publisher: Luiz Miranda / Microsoft Corporation
+PackageName: File Replication Service Diagnostics Tool

```

```suggestion
PackageName: Windows Resource Kit Tools - FRSDiag.exe
```
This should match control panel, or AppsAndFeaturesEntries should be added to the installer manifest